README.md

```diff
@@ -5,6 +5,43 @@ to fully record the interactions
 between assets (files and tools)
 when a software artifact is being built (compiled).
 
+## Synopsis
```

Comment:
Call it "Syntax" or something.

The "Synopsis" is specialized term, and will only be used on the man page.

Reply:
@zvr What about "Usage"?

Reply:
Sure, that's fine.